starting_package/openvino_workbench/arguments_parser.py

```diff
@@ -18,203 +18,246 @@
+                                  required=False,
+                                  help='Specifies the name of the SSL key file. '
+                                       'The file should be placed in the directory provided in the "assets-directory" '
+                                       'argument. '
+                                       'Example: key.pem')
+        self._parser.add_argument('--verify-ssl',
+                                  action='store_true',
+                                  required=False,
+                                  help='Sets the TLS certificate as trusted (\'on\').',
+                                  default=True)
+
+        # DevCloud
+        self._parser.add_argument('--cloud-service-address',
+                                  required=False,
+                                  help='Specifies the URL to a standalone cloud service '
+                                       'that provides Intel(R) hardware capabilities for experiments.')
+        self._parser.add_argument('--cloud-service-session-ttl',
+                                  required=False,
+                                  help='Specifies the cloud service session time to live in minutes.')
+
+        # Network
+        self._parser.add_argument('--network-name',
+                                  required=False,
+                                  help='Specifies the name of a Docker network to run the Docker container in.',
+                                  default='workbench_network')
+        self._parser.add_argument('--network-alias',
+                                  required=False,
+                                  help='Specifies the alias of the DL Workbench container in the network.',
+                                  default='workbench')
+
+        # Misc
+        self._parser.add_argument('--base-prefix',
+                                  required=False,
+                                  help='Specifies the base prefix of the DL Workbench web application.',
+                                  default='/')
+
+    def _validate_arguments(self):
+        self._validate_restart_arguments()
+        self._validate_ssl_arguments()
+        if platform.system() == 'Windows':
+            self._validate_arguments_for_windows()
+
+    def _validate_ssl_arguments(self):
+        self._logger.debug('Validating arguments for SSL.')
+        # Check for SSL-related files
+        if not self.arguments.assets_directory and (self.arguments.ssl_key_name or self.arguments.ssl_certificate_name):
+            self._logger.debug('SSL key and/or certificate were provided without the assets directory.')
+            self._parser.error('ERROR: "--assets-directory" is required for SSL. SSL key and certificate should be '
+                               'placed there.'
+                               '\nExample command:'
+                               f'\n\n\t{CLI_COMMAND} --assets-directory /path/to/assets --ssl-key-name key.pem '
```

Comment:
It would be really great to reuse CLI argument names when printing log messages. It will help a lot to maintain these messages in case of changing arguments names. I would suggest to create a utility class representing `openvino-workbench` tool and keep tool exec command and all its arguments there as constant strings and then reuse it in argument parser and other files where we print usage examples.
However if it look like another big enhancement it can be addressed in separate PR.

Reply:
Wholeheartedly agree yet this PR is huge enough already - I've created a separate ticket for this (78673).